config.toml

```diff
@@ -1,4 +1,4 @@
 [params.plausible]
-   enable      = true
-   subdomain   = false
-   analytics   = "your-plausible-code"
+   enable = true
```

Comment:
If you agree I propose to use those names. imo they avoid confusion between all those "domains".
If it is OK for you, can you adapt your PR ? Thanks again for your contribution. Much appreciated.

```toml
[params.plausible]
   # Whether to enable plausible tracking
   enable            = true
   # This is the plausible "domain" name/Id in your dashboard
   plausible_domain  = "your-plausible-domain-name"
   # Whether to serve the script from a custom domain (Optional : https://docs.plausible.io/custom-domain)
   custom_js_domain  = "stats.example.com"
   # Self-hosted plausible domain (Optional : "plausible.io" is used if unset)
   selfhosted_domain = "myplausible.example.com"

```

Reply:
Have done, although left `domain` as-is as it's still descriptive enough, and avoid duplication of "plausible"